packages/web3torrent/config/jest/fileTransform.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
```

Comment:
Can you add some plain english description to explain why we need this file?

Reply:
All these files are created by running `yarn eject`